src/scripts/index.js

```diff
@@ -973,3 +973,35 @@ $(document).ready(function () {
     }
   })
 })
+
+// timer
+$(document).ready(function () {
+  var countDownDate = new Date('Dec 10, 2018 00:00:00').getTime()
+  var x = setInterval(function () {
+    // Get todays date and time
+    var now = new Date().getTime()
+
+    // Find the distance between now and the count down date
+    var distance = countDownDate - now
+
+    // Time calculations for days, hours, minutes and seconds
+    var days = Math.floor(distance / (1000 * 60 * 60 * 24))
```

Comment:
Define precalculated variables `MINUTE`, `HOUR` and `DAY` and use it like this:

```js
var days = Math.floor(distance / DAY);
var hours = Math.floor((distance % DAY) / HOUR)
```

Reply:
omw